application/templates/static_site/static_pages/ethnic_groups.html

```diff
@@ -16,9 +15,6 @@
 {% block socialDescription %}There are 18 ethnic groups recommended for use by government in England and Wales. Find out how the ethnic groups were chosen, and see how the groups differ in Scotland and Northern Ireland.{% endblock %}
 
 
-{% block headEnd %}<link rel="canonical" href="{{ config.RDU_SITE | strip_trailing_slash }}{{ url_for('static_site.ethnicity_in_the_uk_page', page_name='ethnic-groups-and-data-collected') }}">{% endblock %}
```

Comment:
Maybe we should keep this rel-canonical link, but updated with the new URL? May help with search engine indexing?

Reply:
We don't do it on most pages. And the canonical link would just be itself, right? Which presumably is the default.

Reply:
👍